portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload = "rotateImages()">
+  <body onload = "rotateImages()";"getCommentsUsingArrows()">
```

Comment:
I don't believe this will work as you expect it to. HTML attributes are specified by:

```
<tagname attributename="attributevalue">
```

In this case, the value of `onload` is `rotateImages()`. The value begins after the equals sign and opening double-quote, and ends when a matching double-quote is reached. If you want to execute more than one statement in JavaScript, you can use what's called an ["immediately invoked function expression" (IIFE)](https://developer.mozilla.org/en-US/docs/Glossary/IIFE).

tl;dr:

```
<body onload="(function () {
  rotateImages();
  getCommentsUsingArrows();
})();">